src/shell/scripts/omp.bash

```diff
@@ -78,14 +78,8 @@ function _omp_hook() {
     PS1="$(::OMP:: print primary --config="$POSH_THEME" --shell=bash --shell-version="$BASH_VERSION" --status="$_omp_status_cache" --pipestatus="${_omp_pipestatus_cache[*]}" --execution-time="$_omp_elapsed" --stack-count="$_omp_stack_count" --no-status="$_omp_no_exit_code" --terminal-width="${COLUMNS-0}" | tr -d '\0')"
 }
 
-# rprompt
-function _omp_rprompt() {
-    ::OMP:: print right --config="$POSH_THEME" --shell=bash --shell-version="$BASH_VERSION" --status="$_omp_status_cache" --pipestatus="${_omp_pipestatus_cache[*]}" --execution-time="$_omp_elapsed" --stack-count="$_omp_stack_count" --no-status="$_omp_no_exit_code" --terminal-width="${COLUMNS-0}" | tr -d '\0'
-    return $_omp_status_cache
```

Comment:
This line should be added to the bottom of the `_omp_hook` function (where it is originally located).

Reply:
@lewis-yeung I was unaware it was moved, again, good catch.

Reply:
I remember this because it was moved by me. 😆